mail_attach_existing_attachment_account/wizard/account_invoice_send_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright 2019 Thore Baden
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <record id="account_invoice_send_wizard_form" model="ir.ui.view">
+        <field name="name">Send Invoice</field>
+        <field name="model">account.invoice.send</field>
+        <field name="inherit_id" ref="account.account_invoice_send_wizard_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='attachment_ids']" position="after">
+                <field name="model" invisible="1"/>
+                <field name="res_id" invisible="1"/>
+                <field name="can_attach_attachment" invisible="1"/>
```

Comment:
I'm not sure why, but it looks like `can_attach_attachment` doesn't get set to `True` and so the checkboxes don't show up.
Tested on runbot and on a local machine.
If I remove the `invisible="1"` here and check the checkbox, the attachment checkboxes appear.
Am I missing something?